setup.py

```diff
@@ -8,13 +8,15 @@
 setup(
     name='vidify',
     version=__version__,
-    packages=['vidify'],
+    packages=find_packages(exclude=('tests*', 'images', 'dev*')),
     description='Watch music videos for the songs playing on your device',
     long_description=open('README.md', 'r').read(),
     long_description_content_type='text/markdown',
     url='https://github.com/marioortizmanero/vidify',
     license='LGPL',
 
+    include_package_data=True,
```

Comment:
Why did you add this?

Reply:
That is required for the files in the MANIFEST.in file to be included, without it the file is ingored:
https://python-packaging.readthedocs.io/en/latest/non-code-files.html